spec/system/admin/orders/invoices_spec.rb

```diff
@@ -36,7 +36,7 @@
 
   let(:row1){
     [
-      Time.current.to_date.to_s,
+      I18n.l(Time.current.to_date, format: :long),
```

Comment:
This can fail when you run the spec at midnight. We had this kind of flakiness before and it was very hard to track down.

I would pass the dates to the factories and then test against the resulting string.